dll/win32/imm32/candidate.c

```diff
@@ -161,27 +161,30 @@ ImmGetCandidateListAW(HIMC hIMC, DWORD dwIndex, LPCANDIDATELIST lpCandList, DWOR
     LPCANDIDATELIST pCL;
 
     pClientImc = ImmLockClientImc(hIMC);
-    if (!pClientImc)
+    if (IS_NULL_UNEXPECTEDLY(pClientImc))
         return 0;
 
     uCodePage = pClientImc->uCodePage;
     pIC = ImmLockIMC(hIMC);
-    if (pIC == NULL)
+    if (IS_NULL_UNEXPECTEDLY(pIC))
     {
         ImmUnlockClientImc(pClientImc);
         return 0;
     }
 
     pCI = ImmLockIMCC(pIC->hCandInfo);
-    if (pCI == NULL)
+    if (IS_NULL_UNEXPECTEDLY(pCI))
     {
         ImmUnlockIMC(hIMC);
         ImmUnlockClientImc(pClientImc);
         return 0;
     }
 
     if (pCI->dwSize < sizeof(CANDIDATEINFO) || pCI->dwCount <= dwIndex)
+    {
+        ERR("\n");
```

Comment:
Why?

Reply:
I think `pCI->dwSize < sizeof(CANDIDATEINFO)` is a logical error.
And `pCI->dwCount <= dwIndex` is a boundary check.

Reply:
I meant, why not add anything in the error print message.

Reply:
The program size bloating is a severe problem.
I thought this message was enough for understanding.

Reply:
Since when are a few debug messages a severe problem?

Reply:
Do you want to 'normalize' all the comments in `IMM32`?

Reply:
Do you want to 'normalize' all the trace messages in `IMM32`?

Reply:
I am interested in the reasoning behind 'debug messages are a problem'

Reply:
https://github.com/reactos/reactos/pull/4807